server/encryption/localkey_handler.go

```diff
@@ -0,0 +1,217 @@
+package encryption
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"io"
+	"os"
+
+	"github.com/google/tink/go/kwp/subtle"
+)
+
+const (
+	// DataKeyLength provides the length for data key in bytes.
+	// It is recommended to use an authentication key with 32 or 64 bytes.
+	// The data key length must be either
+	// 16, 24, or 32 bytes to select AES-128, AES-192, or AES-256 modes.
+	DataKeyLength int = 32
+)
+
+var (
+	masterKeyVarName = "LOCAL_MASTER_KEY"
```

Comment:
I'd also make this a const and call it `MasterKeyEnvVarName`.

Reply:
wip